tests/transchunk_manyins.test/log.expected

```diff
@@ -1,3 +1,3 @@
 Starting coproc
-(out='    OSQL_DONE            49999')
+(out='    OSQL_DONE_SNAP       50000')
```

Comment:
Test does 100K write in a chunk size of 2. The osql counters here used to be 49999 OSQL_DONE + 1 OSQL_DONE_SNAP. The 49999 OSQL_DONE were chunks committed in the middle of the client transaction. The 1 OSQL_DONE_SNAP was the final chunk that committed via the normal sqlite3btreecommit code path. Now, In this PR, all chunks will send snapinfo to leader so they can get back query effects, hence we should see 50K OSQL_DONE_SNAP's here.

Reason for the opcode change is because, test does a 'send stat long' and grep for table with with an `-A 3` option to also grab the next 3 lines after the table, and check the number of "DONE"'s. So imagine this output:

table t
OSQL_USEDB 50000
OSQL_INSERT 100000
OSQL_DONE 49999
OSQL_DONE_SNAP 1

It would expect "OSQL_DONE 49999".

Again, in this PR, the output would look something like this:

table t
OSQL_USEDB 50000
OSQL_INSERT 100000
OSQL_DONE_SNAP 50000

So test expected output should be adjusted to "OSQL_DONE_SNAP 50000".